charts/icm-as/.bumpversion.cfg

```diff
@@ -0,0 +1,13 @@
+[bumpversion]
+current_version = 0.2.6
+commit = true
+tag = false
+message = Bump icm-as chart version: {current_version} → {new_version}
+
+[bumpversion:file:Chart.yaml]
+search = version: {current_version}
+replace = version: {new_version}
+
+[bumpversion:file:README.md]
```

Comment:
That `README.md` section is only necessary if the current version is mentioned in that file. Which is not the case so you can safely remove this block.

Reply:
done